clirr-ignores.xml

```diff
@@ -147,4 +147,11 @@
         <justification>False positive, the enclosing class is package-private so this was never exposed</justification>
     </difference>
 
+    <difference>
```

Comment:
I think this is ok?  The method was protected, so in theory classes that extend `BuiltStatement` could access it, but you can't extend `BuiltStatement` directly since its abstract method `buildQueryString` is package private.  I guess you could extend a subclass, but it isn't clear which one you could extend since they all seem to have package private constructors.